api/scripts/organization/update-organization-province-code.js

```diff
@@ -0,0 +1,93 @@
+import * as url from 'url';
+
+import _ from 'lodash';
+import bluebird from 'bluebird';
+
+import { checkCsvHeader, parseCsvWithHeader } from '../helpers/csvHelpers.js';
+import { disconnect } from '../../db/knex-database-connection.js';
+import { updateOrganizationProvinceCode } from '../../lib/domain/usecases/organizations-administration/update-organization-province-code.js';
+import * as organizationForAdminRepository from '../../lib/infrastructure/repositories/organization-for-admin-repository.js';
+
+const modulePath = url.fileURLToPath(import.meta.url);
```

Comment:
Suggestion :
* Utiliser le protocole `node:` pour les modules _built-in_ pour plus de clarté cf. https://2ality.com/2021/12/node-protocol-imports.html
* Utiliser `fileURLToPath` directement sans utiliser le namespace du package tellement la fonction est explicite.

```suggestion
import { fileURLToPath } from 'node:url';

import _ from 'lodash';
import bluebird from 'bluebird';

import { checkCsvHeader, parseCsvWithHeader } from '../helpers/csvHelpers.js';
import { disconnect } from '../../db/knex-database-connection.js';
import { updateOrganizationProvinceCode } from '../../lib/domain/usecases/organizations-administration/update-organization-province-code.js';
import * as organizationForAdminRepository from '../../lib/infrastructure/repositories/organization-for-admin-repository.js';

const modulePath = fileURLToPath(import.meta.url);
```